src/Coduo/PHPHumanizer/String/ShortcodeProcessor.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Coduo\PHPHumanizer\String;
+
+use Thunder\Shortcode\HandlerContainer\HandlerContainer;
+use Thunder\Shortcode\Parser\RegularParser;
+use Thunder\Shortcode\Processor\Processor;
+use Thunder\Shortcode\Shortcode\ShortcodeInterface;
+
+class ShortcodeProcessor
+{
+    public function __construct()
+    {
```

Comment:
@norzechowicz I think that explicit constructor is better, even if it's empty. Do you want me to remove it or can you live with it? Do you have any other issues with this PR?